src/pages/signup/index.tsx

```diff
@@ -18,8 +18,9 @@ const SignUp = () => {
   const [email, setEmail] = useState<string>('');
   const [nickname, setNickname] = useState<string>('');
   const [password, setPassword] = useState<string>('');
-  const [error, setError] = useState<boolean>(false);
   const [passwordConfirm, setPasswordConfirm] = useState<string>('');
+  const [emailError, setEmailError] = useState<boolean>(false);
+  const [signupSuccess, setSignupSuccess] = useState<boolean>(false);
```

Comment:
```suggestion
  const [emailErrorCatched, setEmailErrorCatched] = useState<boolean>(false);
  const [signupSuccessed, setSignupSuccessed] = useState<boolean>(false);
```

boolean값에 걸맞는 네이밍 제안드립니다!